src/sha1.ts

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable */
+/**
+ * Copied from https://github.com/pH200/hmacsha1-js/blob/master/index.js
```

Comment:
Can you also paste the full copyright message here? Would be appropriate I belief (for when the link is dead for instance)

Reply:
Done ✅